lib/ReactViews/RelatedMaps/related-maps.scss

```diff
@@ -1,7 +1,13 @@
+@import "~terriajs-variables";
+
 .dropdown-inner {
-  width: 600px;
   padding: 10px;
   font-size: 0.9rem;
+  width: 600px;
+
+  @media (max-width: $mobile) {
+    max-width: fit-content;
+  }
```

Comment:
I had a look but couldn't figure out the change, a screenshot would be good.

Reply:
This was a git fail on my behalf this commit relates to this issue https://github.com/TerriaJS/terriajs/issues/6633, when I created the branch I branched off the feature branch not main